src/components/day/Day.js

```diff
@@ -628,7 +634,16 @@ export default class DayComponent extends Field {
     }
     const [DAY, MONTH, YEAR] = this.component.dayFirst ? [0, 1, 2] : [1, 0, 2];
     const values = value.split('/');
-    return (values[DAY] === '00' || values[MONTH] === '00' || values[YEAR] === '0000');
+   // return (values[DAY] === '00' || values[MONTH] === '00' || values[YEAR] === '0000');
+
+
+
+    return (values[DAY] === '00' ||
+      values[MONTH] === '00' ||
+      values[YEAR] === '0000' ||
+      values[DAY] === '' ||
+      values[MONTH] === '' ||
+      values[YEAR] === '');
   }
 
   getValidationFormat() {
```

Comment:
Does this method still work when some part of date is hidden?

Reply:
fixed